src/main/java/com/github/manolo8/darkbot/utils/LogUtils.java

```diff
@@ -24,13 +26,27 @@ public static void setOutputToFile() {
         if (!Files.exists(LOG_FOLDER)) createFolder();
         else removeOld();
 
-        try {
+        /*try {
             PrintStream output = getLogger();
             System.setOut(output);
             System.setErr(output);
         } catch (FileNotFoundException | UnsupportedEncodingException e) {
             System.out.println("Failed to redirect logs, file not found:");
             e.printStackTrace();
+        }*/
+        try {
+            FileOutputStream fileOut = new FileOutputStream("logs/" + START_TIME + ".log");
```

Comment:
This is not equivalent to the current behavior, log modifications aren't being appiled here for file output, see getLogger method